jwst/extract_1d/tests/test_apply_apcorr.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+import numpy as np
+import os
+
+from astropy.io import fits
+from astropy.table import Table
+
+from ...datamodels import DataModel
+
+from ..apply_apcorr import ApCorr, ApCorrRadial, ApCorrPhase, select_apcorr
+
+data_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data'))
+NIR_TEST_FILES = {
+    'ISU/MSASPEC': os.path.join(data_dir, 'jwst_nirspec_apcorr_0003.fits'),
+    'FIXEDSLIT/BRIGHTOBJ': os.path.join(data_dir, 'jwst_nirspec_apcorr_0004.fits')
+}
+
+
+@pytest.fixture(
+    scope='module',
+    params=[
+        ('MIRI', 'MIR_LRS-FIXEDSLIT'),
+        ('MIRI', 'MIR_LRS-SLITLESS'),
+        ('MIRI', 'MIR_MRS'),
+        ('NIRCAM', 'NRC_GRISM'),
```

Comment:
If this is for the NIRCam time series grism mode, then it should be "NRC_**TS**GRISM"

Reply:
Same as the other comment above. I think the correct answer is to remove it